app/uk/gov/hmrc/traderservices/controllers/ContactFieldHelper.scala

```diff
@@ -38,8 +43,15 @@ object ContactFieldHelper {
     }
 
   def normaliseNumber(phoneNum: String): String =
-    if (phoneNum.startsWith("+44")) phoneNum.replace("+44", "0").replaceAll("\\s", "")
```

Comment:
This be a better implementation? This way if we need to extend the prefix we can just add it to the list and no further changes will be needed.

    val prefixes = List("+44", "44", "0044")
    prefixes.find(phoneNum.startsWith(_)) match {
      case Some(n) => removeNonAllowedCharacters(phoneNum.replace(n, "0"))
      case None    => removeNonAllowedCharacters(phoneNum)
    }